packages/idyll-compiler/test/test.js

```diff
@@ -261,17 +270,17 @@ describe('compiler', function() {
   });
 
   describe('parser', function() {
-    it('should parse a simple string', function() {
+    it('should parse a simple string', async function() {
       var input = 'Just a simple string';
-      expect(compile(input, { async: false })).to.eql(
+      expect(await compile(input)).to.eql(
```

Comment:
Is the move to the async compiler call just stylistic / convenience? Or is there some functional update here

Reply:
All compile calls return Promises now (all are async), which IMO improves consistency and prevents future errors (e.g., if a caller was using a sync approach but then added an async plugin). So the `await` call (or a `.then`) is necessary for correctness.